docs/tutorials/03_shaders/viz_sdf_cylinder.py

```diff
@@ -0,0 +1,296 @@
+"""
+===============================================================================
+Make a Cylinder using polygons vs SDF
+===============================================================================
+This tutorial is intended to show two ways of primitives creation with the use
+of polygons, and SDFs. We will use cylinders as an example since they have a
+simpler polygonal representation. Hence, it allows us to see better the
+difference between using one or the other method.
+
+For the cylinder representation with polygons, we will use cylinder actor
+implementation on FURY, and for the visualization using SDFs, we will
+implement shader code to create the cylinder and use a box actor to put our
+implementation inside.
+
+We start by importing the necessary modules:
+"""
+
+from fury import actor, window
+from fury.shaders import compose_shader, shader_to_actor, attribute_to_actor,\
+    import_fury_shader
+
+import os
+import numpy as np
+
+###############################################################################
+# Cylinder using polygons
+# ================
```

Comment:
too short, should be the same size a s the title, warning appear during doc generation